ibis/expr/tests/test_api.py

```diff
@@ -124,3 +125,16 @@ def test_duplicate_columns_in_memtable_not_allowed():
 
     with pytest.raises(IbisInputError, match="Duplicate column names"):
         ibis.memtable(df)
+
+
+def test_implicit_coercion_of_null_literal():
+    # GH #7775
+    expr1 = ibis.literal(True) & ibis.null()
+    expr2 = ibis.literal(True) & None
+
+    expected = ops.And(
```

Comment:
How about a test for the operation from the issue title (Or), as well as Xor?

Reply:
Going to parametrize it

Reply:
Parametrized with AND OR XOR